armi/reactor/tests/test_reactors.py

```diff
@@ -812,7 +812,10 @@ def test_createAssemblyOfType(self):
         # basic creation
         aOld = self.r.core.getFirstAssembly(Flags.FUEL)
         aNew = self.r.core.createAssemblyOfType(aOld.getType(), cs=self.o.cs)
-        self.assertAlmostEqual(aOld.getMass(), aNew.getMass())
+        self.assertAlmostEqual(
```

Comment:
Why is this change necessary?

Reply:
the fresh assembly is not located in the core so its values are "full"